integration-test/init_test.go

```diff
@@ -23,6 +25,7 @@ const (
 	initrdName     = "hyper-initrd.img"
 	binaryName     = "runv"
 	rootfsName     = "rootfs"
+	logFileName    = "runv.INFO"
```

Comment:
the runv.ERROR is not printed in this patch.
if the test starts multiple runv daemon, it prints the last one only.